spec/ddtrace/method_tracing_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'ddtrace'
+
+module Fake
```

Comment:
We normally avoid creating top level constants during tests, to avoid polluting the namespace for subsequent tests.

For temporary classes we normally follow a pattern similar to:
https://github.com/DataDog/dd-trace-rb/blob/dfe97dbea117c987e53c22bf2a9cffc4707c9dfc/spec/ddtrace/analytics_spec.rb#L59-L69